spec/features/user_places_order_spec.rb

```diff
@@ -5,15 +5,12 @@
   fixtures :stores
   fixtures :photos
 
-  let!(:user) {User.first}
-  let!(:photo) {Photo.first}
+  let!(:user) { User.first }
+  let!(:photo) { Photo.first }
 
   before do
-    visit root_path
-
-    within(".popular-photographs") do
-      first(:button, "Add to Cart").click
-    end
+    visit photos_path
+    first(:button, "Add to Cart").click
```

Comment:
fyi, at some point the id got removed, but it works fine just selecting the first Add to Cart